src/components/Launcher.js

```diff
@@ -13,6 +13,15 @@ class Launcher extends Component {
       active: false,
       launcherIcon
     };
+    this.handleClick = this.handleClick.bind(this);
+    this.openChat = this.openChat.bind(this);
+  }
+
+  //developer may pass in a boolean called toggle  and toggle it to open chat window
+  componentDidUpdate(prevProps, prevState) {
```

Comment:
We don't need this comment. People will be able to deduce this by reading the code and this comment may quickly become inaccurate or misleading.